shared/src/common/enums/validation/password-validation.enum.ts

```diff
@@ -0,0 +1,7 @@
+enum PasswordValidation {
+  MIN_LENGTH = 6,
+  MIN_LENGTH_MESSAGE = 'Password must have at least 6 characters',
+  REQUIRED_MESSAGE = 'Password is required',
+}
```

Comment:
We should't create a new validation enum for each form-field. It should be one per each form
Also, it would be better have one for `rules` and one for messages
Example:
![image](https://user-images.githubusercontent.com/60396171/110286319-6695c480-7fed-11eb-807f-bd40f8ef3fb3.png)
![image](https://user-images.githubusercontent.com/60396171/110286346-701f2c80-7fed-11eb-9551-3894f7a38f1a.png)